### ISSUE TITLE
Using offline payment throws exception

### ISSUE THREAD

**Issue by torinaki** [Sylius/Sylius#3685]
Faced with following exception on order submit:
```
Gateway "offline" does not exist.
```
Seems new version of Payum bundle do not have such gateway configured:
```
$ php ./app/console debug:config payum
payum:
    gateways:
        paypal_express_checkout: { paypal_express_checkout_nvp: { username: db-facilitator_api1.zb.lv, password: 1386079449, signature: AA.EdtBRYFrtdnjw9s-2azOre0X4AYbckaCSL.pubIbuKwTqOX0owTYi, sandbox: true, actions: {  }, apis: {  }, extensions: {  } } }
        be2bill_direct: { be2bill_direct: { identifier: EDITME, password: EDITME, sandbox: true, actions: {  }, apis: {  }, extensions: {  } } }
        be2bill_offsite: { be2bill_offsite: { identifier: EDITME, password: EDITME, sandbox: true, actions: {  }, apis: {  }, extensions: {  } } }
        stripe_checkout: { stripe_checkout: { publishable_key: EDITME, secret_key: EDITME, actions: {  }, apis: {  }, extensions: {  } } }
        dummy: { offline: { actions: {  }, apis: {  }, extensions: {  } } }

```

**Comment by Sp0ng3b0b** [comment]
payum:
    gateways:
        offline:
            offline: ~

Add this to your configuration, works for me :+1:

**Comment by michalmarcinkowski** [comment]
If it didn't work for you please post this question on StackOverflow - it is the official support channel for Sylius. Just tag your question with "sylius" and "payum", maybe someone will be able to help you! http://stackoverflow.com/questions/tagged/sylius

**Comment by torinaki** [comment]
@Sp0ng3b0b thanks you. I will try. More than sure it will work.

@michalmarcinkowski It is not a question and even not Support Request. Currenlty `offline` payment type do not work in default Sylius setup, because of incomplite payum setup(I guess it happen after Payum bundle update till 1.0 version). I have faced with this issue right after Sylius update.
The only question is what should be fixed to solve this issue:
1. Remove offline payment type from Sylius setup
2. Payum configuration should be adjusted as @Sp0ng3b0b wrote.

// cc @pjedrzejewski

**Comment by pjedrzejewski** [comment]
@torinaki Your solution sounds good, are you willing to work on a patch?

**Comment by torinaki** [comment]
Yes, sure. I will prepare PR to adjust default configuration.
On Ср, 9 дек. 2015 г. at 20:42, Paweł Jędrzejewski <notifications@github.com>
wrote:

> @torinaki <https://github.com/torinaki> Your solution sounds good, are
> you willing to work on a patch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sylius/Sylius/issues/3685#issuecomment-163353332>.
>
-- 
Best Regards,
Dmitrijs Balabka
Software Architect

E-Mail: dmitry.balabka@gmail.com
Cell Phone: +371 29671519
Web: http://dmitry.balabka.com/

**Comment by michalmarcinkowski** [comment]
Ok, so it's a bug then. @torinaki thanks for reporting, looking forward to PR :wink:

**Comment by Sp0ng3b0b** [comment]
I dont know how others are using offline payment but i needed to be able to approve the payment manually. In the default manual gateway its automatically set to payment finished, i changed the payment gateway to set the payment state after checkout to processing, making it possible to approve it manually. If someone needs that functionality let me know :)

**Comment by makasim** [comment]
> I dont know how others are using offline payment but i needed to be able to approve the payment manually.

You have to provide a custom converter which replaces this one https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/PayumBundle/Action/ConvertPaymentToOfflineAction.php#L23

**Comment by Sp0ng3b0b** [comment]
@makasim exactly what i did, i was just asking if someone needs that fix ;)

**Comment by pjedrzejewski** [comment]
Fixed in #3995.
